### ISSUE TITLE
Tough to read years from the main plot when showing multiple years.

### ISSUE THREAD

**Issue by KristofferStrube** [dotnet/nuget-trends#181]
# Problem
When plotting the downloads for a package over multiple years we have a `'MMM d'` format as the label for the x-axis and the same for the tooltip title when hovering over a specific data-point. This can make it tough to read which year a specific data point is for.

# Solutions
## Simple solution
The most simple solution that would require minimal changes would be to set the title of the tooltip explicitly to include the year.
This can be done [here in packages.ts](https://github.com/dotnet/nuget-trends/blob/main/src/NuGetTrends.Web/Portal/src/app/packages/packages.component.ts#L135) by adding a title property. This keeps the plot simple on the surface but makes it possible to inspect the full date if you can't figure out the year.
## Less simple solution
Have the x-axis labels depend on whether or not we are plotting for 1 year or more. I can't figure out specifically where this should be done but should be doable with more changes to the packages.ts code. I'm not sure how pretty this will be code-quality-wise.

**Comment by bruno-garcia** [comment]
Agreed! This is something that I always noticed too but never got around to change.

I personally don't have an opinion on which route to take as a solution (@joaopgrassi might have one). But happy to review a PR and get this changed and released.

**Comment by joaopgrassi** [comment]
@KristofferStrube if you have time to send a PR, we can go with the simple solution first and see how it looks.

**Comment by joaopgrassi** [comment]
Hum strange. I guess you used the contrib db from here: https://github.com/dotnet/nuget-trends#database-backup-for-contributors?

I haven't use it in a while, but last time I did it worked. My "go-to" packages to try-out things are usually the popular ones like `EntityFramework`, `Dapper` or `xUnit` and the like. Maybe try one of those? 

If I have time this week I'll try setting up from scratch to see if I run into problems.

**Comment by KristofferStrube** [comment]
Yep @joaopgrassi I used that backup.

I will try to look at using one of those packages to check the import again.

And thanks. There were some hoops to jump through to get it to work without prior setup, but I think I got there.

**Comment by bruno-garcia** [comment]
This was fixed by a contributor PR a while back
